src/parser.cc

```diff
@@ -1074,11 +1074,12 @@ void addNestedModel(ElementPtr _sdf, ElementPtr _includeSDF)
   std::string modelName = modelPtr->Get<std::string>("name");
   while (elem)
   {
+    std::string elemName = elem->Get<std::string>("name");
```

Comment:
This will rename all elements with the `name` attribute. That might break things (eg. `<plugin>` tags), so I suggest we limit it to `link`, `joint`, and `model`.

Reply:
good catch, added 579da3b